include/deal.II/multigrid/mg_smoother.h

```diff
@@ -467,6 +467,15 @@ class MGSmootherPrecondition : public MGSmoother<VectorType>
              const typename PreconditionerType::AdditionalData &
                additional_data = typename PreconditionerType::AdditionalData());
 
+  /**
+   * In contrast to the function above, only initialize the matrices. The
+   * smoothers need to be setup manually by the user in the following. This
+   * is useful if one wants full flexibility in the choice of smoothers.
```

Comment:
Maybe you can explain a bit more how this works: By manually setting the public member variable `smoothers`?